src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,20 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import mate.academy.spring.validator.Email;
+import mate.academy.spring.validator.RepeatPassword;
+
+@RepeatPassword
 public class UserRequestDto {
+    @Email
     private String email;
+    @NotNull
+    @Min(8)
```

Comment:
Read the documentation for @Min annotation. It's for numbers only. 
Use @Size(min = 8)

Reply:
done